### ISSUE TITLE
Accessibility update to notebook.html missing alt text

### ISSUE THREAD

**Issue by snidersd** [jupyter/notebook#3226]
Also see: [jupyter/notebook#1801](https://github.com/jupyter/notebook/issues/1801)
After running the Dynamic Assessment Plugin for Chrome and the Digital Content Checker on notebook.html the following issue was found and can be corrected as indicated:
Line 48 requires alt text - `<img class="current_kernel_logo" alt="Current Kernel Logo" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"/>`
If the image is not important then include alt="".

**Comment by ns23** [comment]
Can I work on this issue?

**Comment by takluyver** [comment]
Go for it! Can you see what to do?

**Comment by ns23** [comment]
I have to add `alt="Current Kernel Logo"` on the line below.
https://github.com/jupyter/notebook/blob/6526a19c0b61137fdb3e4bf9406c8e026a3043fd/notebook/templates/notebook.html#L49
Please correct me If I am wrong.

**Comment by takluyver** [comment]
Sounds right, thanks!

**Comment by ns23** [comment]
:slightly_smiling_face:   okay I will start working on it and create PR asap

**Comment by ns23** [comment]
@takluyver  I have created the PR.Please review it.

**Comment by takluyver** [comment]
Thanks @ns23

**Comment by ns23** [comment]
Welcome. I am looking forward to contribute more to this project ☺️
